cypress/objects/timelog.js

```diff
@@ -6,7 +6,23 @@ cy.url().should('include', '/time-logging');
 pagetitle(){
 return cy.get('.page__title')
 }
-
+calendarv(){
+return cy.get('.calendar')
+}
+getTile(){
+    return cy.get('.tile.form');
+}
+getUserTitle(){
+    return cy.get('.user-info__title');
+}
+mainNav(){
```

Comment:
The function name tells us nothing because it's not descriptive. If this function is meant to retrieve all the tabs from the main navigation bar, then rename it to something like "`getNavigationTabs()`"